src/one_dragon/base/cv_process/cv_service.py

```
@@ -181,39 +224,109 @@ def _emit_overlay_vision(self, pipeline_name: str, context: CvPipelineContext) -
+        for file_name in os.listdir(dir_path):
             if file_name.endswith('.yml'):
                 names.append(file_name[:-4])
         return names
 
-    def save_pipeline(self, name: str, pipeline: CvPipeline) -> bool:
+    def save_pipeline(self, name: str, pipeline: CvPipeline, source: str | None = None) -> bool:
         """
         将流水线保存到文件
         :param name: 流水线名称
         :param pipeline: 流水线实例
+        :param source: 保存目标来源；None 表示主仓，插件名表示对应插件目录
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF002)